ui/components/Link.tsx

```diff
@@ -24,6 +32,10 @@ function Link({
   textProps,
   ...props
 }: Props) {
+  if (href && !isAllowed(href)) {
+    return <Text {...textProps}>{children}</Text>;
+  }
+
```

Comment:
I think it'd be useful to do the thing here where you also pass the textProps down - that way we can still style this Text component the way we want. Just spread the textProps thing this component already has like line 40. It was a quick fix I did a long time ago for something and forgot about but here it is! There might be a better way.